doc/content/examples/figures/divertor_monoblock_sensitivity_figures/tmap_processing_v1.py

```diff
@@ -0,0 +1,1190 @@
+# DETAILS: N. H. Snow Rev1
+# Good luck to anyone who will attempt to edit or use this file.
+
+from matplotlib import pyplot as plt; from matplotlib.colors import TABLEAU_COLORS as plt_colors
+import pandas as pd; from pandas import DataFrame
+import numpy as np; from numpy import ndarray
+import typing; from typing import Union
+import json
+import os
+from importlib import reload
+from scipy import optimize; import scipy.stats as scipy
+import seaborn
+from IPython.display import HTML
+
+plt = reload(plt)
+plt.rcParams['font.family'] = 'Cambria'
+plt.rcParams['font.size'] = 11
+colors = list(plt_colors.values())
+
+def json_serial_killer(an_obj, **kwargs):
```

Comment:
There is a some overlap between this file and `sobol_processing.py` Could we combine them, or maybe create a third python script that does everything that is in common.